cypress/integration/nav_bar.ts

```diff
@@ -100,4 +104,31 @@ describe("Nav Bar", () => {
     );
     cy.getCookie(projectCookie).should("have.property", "value", "spruce");
   });
+
+  describe("Admin settings", () => {
+    it("Should not show Admin button to non-admins", () => {
+      const userData = {
+        data: {
+          user: {
+            userId: "admin",
+            displayName: "Evergreen Admin",
+            emailAddress: "admin@evergreen.com",
+            permissions: {
+              canEditAdminSettings: false,
+            },
+          },
+        },
+      };
+      cy.overwriteGQL("User", userData);
+      cy.visit(SPRUCE_URLS.version);
+      cy.dataCy("admin-link").should("not.exist");
+    });
```

Comment:
As much as I'm not a fan of mocking requests in e2e tests, I understand why. I think this is a good motivation for us to begin thinking about having multiple users for e2e tests in the future. It will allow us to more accurately test permission things.